paket.dependencies

```diff
@@ -1,22 +1,50 @@
-source http://nuget.org/api/v2
-source https://ci.appveyor.com/nuget/sourcelink-helmesfwai2a
-
-redirects: on
-
-nuget FSharp.Core 3.1.2.5
-nuget Zlib.Portable
-nuget FSharp.Charting
-nuget FSharp.Formatting
-nuget NUnit
-nuget NUnit.Runners
-nuget FsUnit 2.3.2
-nuget FsCheck
-nuget Nancy.Hosting.Self
+framework: >=net45
+source https://api.nuget.org/v3/index.json
+
+# For testing against .NET Standard 2.0
+nuget NetStandard.Library.NetFramework
+# For testing against .NET Standard 2.0
+nuget Microsoft.NETCore.App
+
 nuget FAKE
-nuget SourceLink.Fake
 nuget Nuget.CommandLine
 
 github fsharp/FAKE modules/Octokit/Octokit.fsx
+
 github fsprojects/FSharp.TypeProviders.StarterPack src/ProvidedTypes.fsi
 github fsprojects/FSharp.TypeProviders.StarterPack src/ProvidedTypes.fs
 github fsprojects/FSharp.TypeProviders.StarterPack src/ProvidedTypesTesting.fs
+
+# NOTE: These are only used for advice by examining the versions chosen in the paket.lock file.
+# NOTE: The PackageReference items are themselves added by hand to the project files.
+# NOTE: This is because paket's diagnostics in missing packages were not helpful for .NET SDK cases.
+# NOTE: We can return to using paket.references files at a later point.
+group Test
+    frameworks: netcoreapp2.0, net46
+    source https://api.nuget.org/v3/index.json
+    redirects: on
+
+    nuget FSharp.Core
+    nuget NUnit
+    nuget NUnit.Runners
+    nuget FsUnit
+    nuget FsCheck
+    nuget FSharp.Charting
+    nuget FSharp.Formatting
+    nuget Nancy.Hosting.Self
+
+# For testing against FSharp.Core for F# 3.1
+group fs31
+    source https://api.nuget.org/v3/index.json
+    nuget FSharp.Core ~> 3.1.0
+
+# For testing against FSharp.Core for F# 4.0
```

Comment:
I see these groups defined here, but I can't see what makes the tests run against all these different F# versions

Reply:
Sorry - it comes from here: https://github.com/fsprojects/FSharp.TypeProviders.SDK/blob/master/src/ProvidedTypesTesting.fs#L681

I've added that as a comment
```
. The TPSDK testing helpers like these to be placed in package groups
# with these names, see https://github.com/fsprojects/FSharp.TypeProviders.SDK/blob/0c0c9d11135039821a380f60ec21c701354e8cdb/src/ProvidedTypesTesting.fs#L680
```